lib/redox/media/new.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "redox"
+
+module Redox
+  module Media
+    class New < Redox::Query
+      redox_property :StartDateTime, coerce: ->(v) {
+        v.respond_to?(:iso8601) ? v.iso8601 : v.to_s
+      }
+      redox_property :EndDateTime, coerce: ->(v) {
+        v.respond_to?(:iso8601) ? v.iso8601 : v.to_s
+      }
```

Comment:
Why does the Media::New message have StartDateTime and EndDateTime properties?

I ask because I don't see those properties in the sample here: https://developer.redoxengine.com/data-models/Media.html#New

Reply:
Good catch. It should not, that is copy pasta.